php_apc.c

```diff
@@ -498,9 +502,8 @@ static void apc_store_helper(INTERNAL_FUNCTION_PARAMETERS, const zend_bool exclu
 /* {{{ proto bool apcu_enabled(void)
 	returns true when apcu is usable in the current environment */
 PHP_FUNCTION(apcu_enabled) {
-	if (zend_parse_parameters_none() == FAILURE) {
-		return;
-	}
+	ZEND_PARSE_PARAMETERS_START(0, 0)
+	ZEND_PARSE_PARAMETERS_END();
```

Comment:
PHP 7.1 and below are already EOL and using ZPP_NONE seems appropriate.

```suggestion
	ZEND_PARSE_PARAMETERS_NONE();
```

Reply:
If you want to continue to maintain compatibility with PHP 7.0.0-dev, I believe you can define `ZEND_PARSE_PARAMETERS_NONE` properly using an approach like `php74_shim.h`. As well as `ext/skeleton`.

Reply:
I think we can just keep using zend_parse_parameters_none(), there is no advantage to the FastZPP API in this case.

Reply:
I've changed all the places with ZEND_PARSE_PARAMETERS_START(0, 0) back to zend_parse_parameters_none()